src/pages/login/index.tsx

```diff
@@ -1,41 +1,53 @@
 import React, { useCallback, useEffect } from 'react';
 import { useDispatch } from 'react-redux';
 import { useHistory } from 'react-router-dom';
-import { Center } from '~/components/Box';
 import { getDashboardPath } from '~/constants/paths';
 import { useSelector } from '~/store/hooks';
 import { login } from '~/store/slices/CreateUserSlice';
+import { LoginLayout } from './layout';
+import { LoginForm, formData } from './components/LoginForm';
+import styled from 'styled-components';
+import { Center } from '~/components/Box';
+import Logo from '~/components/Logo';
+import { COLORS } from '~/styles/variables';
+import { defaultTheme } from '~/styles/theme';
 
-export const LoginPage: React.FC = () => {
-  const { isLoading, isAuthenticated, isError } = useSelector(state => state.user);
+const Page: React.FC = () => {
+  const { isAuthenticated } = useSelector(state => state.user);
   const history = useHistory();
   const dispatch = useDispatch();
 
   useEffect(() => {
     if (isAuthenticated) history.push(getDashboardPath());
   }, [isAuthenticated]);
 
-  const onSubmit = useCallback((event: React.FormEvent<HTMLFormElement>) => {
-    event.preventDefault();
-    const form = event.target;
-
-    const username: string = form[0].value;
-    const password: string = form[1].value;
-
-    dispatch(login({ username, password }));
+  const onSubmit = useCallback((data: formData) => {
+    dispatch(login(data));
   }, []);
 
   return (
-    <Center height="100vh" flexDirection="column">
-      <form onSubmit={onSubmit}>
-        <input type="text" name="username" placeholder="Username" />
-        <input type="password" name="password" placeholder="Password" />
-        <button type="submit"> SEND </button>
-        {isError && <h5>ERROR!!!</h5>}
-        {isLoading && <h5>Loading....</h5>}
-      </form>
-    </Center>
+    <LoginLayout>
+      <Center height="100%">
+        <Center
+          flexDirection="column"
+          backgroundColor={COLORS.white}
+          borderRadius="16px"
+          padding={defaultTheme.margins.medium}>
+          <Logo />
+          <LoginForm onSubmit={onSubmit} />
+        </Center>
+      </Center>
+    </LoginLayout>
   );
 };
 
-export default LoginPage;
+export default styled(Page)`
+  > div {
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    max-height: 60%;
+    max-width: 60%;
+  }
+`;
```

Comment:
This doesn't look good... If come `div` inside this page should have the following styles it should be refactored into separate styled components and applied where needed. Such usage of > selector for all divs inside the component can lead to hard to trace bugs and in general doesn't fit to styled-components "way"